tests/unit/client/ssh/test_shell.py

```diff
@@ -0,0 +1,31 @@
+import os
+import subprocess
+
+import salt.client.ssh.shell as shell
+from tests.support.runtests import RUNTIME_VARS
+from tests.support.unit import TestCase
+
+
```

Comment:
Decorate this class with the requires system binaries helper, the windows failures are valid